samples/cloud_run_jobs/large_table/README.md

```diff
@@ -2,30 +2,28 @@
 
 This is an example of distributed DVT usage using [Cloud Run Jobs](https://cloud.google.com/run/docs/create-jobs). Distributed DVT usage alleviates memory constraints when running large table validations. In this sample, you will first generate table partitions with the `generate-table-partitions` command, which will partition your large table into smaller pieces with filters. Read more on generating partitions [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator?#generate-table-partitions-for-large-table-row-validations).The Cloud Run Job can then distribute each partition's YAML configuration as a Cloud Run Task in parallel.
 
-## Quickstart
-
-### Build a Docker Image
+## Build a Docker Image
 
 You will need to build a Docker image to be used by your Cloud Run Job with DVT installed.
 
 ```
-export PROJECT_ID=<PROJECT-ID>
+PROJECT_ID=<PROJECT-ID>
```

Comment:
```suggestion
export PROJECT_ID=<PROJECT-ID>
```
I would bring the `export` part again because it's already used this way 5 times on this specific file. For consistency purposes, it'd be the case of choosing one syntax or the other for all occurrences instead of mixing them.

Reply:
I've chosen to remove other redundant `export` statements. Only the mandatory `export` remains.